keras/engine/training.py

```diff
@@ -928,6 +929,21 @@ def run_eagerly(self):
     def run_eagerly(self, value):
         self._run_eagerly = value
 
+    @property
+    def distribute_reduction_method(self):
+        """Settable attribute indicating how the model should reduce
+        loss and metric values from replicas.
+
+        Default: `"auto"`. This should be good for all cases.
```

Comment:
Suggest: all cases to be replaced by "general use cases"

Reply:
https://github.com/keras-team/keras/pull/16664/commits/db2aba0168d5481e6c78cb90f5652a116b0ada40